rules/aip0203/field_behavior_required.go

```diff
@@ -23,47 +23,62 @@ import (
 	"github.com/jhump/protoreflect/desc"
 )
 
-var minimumRequiredFieldBehavior = stringset.New(
+var fbs = stringset.New(
```

Comment:
fbs doesn't represent all field behaviors (e.g. unordered_list). The naming of this variable was intentional to indicate the required fbs.

If the variable indicates that it's only the required set, I'd be fine with shortening the name.

Reply:
I think it's implied by the nature of the rule and the docs in the md, but I don't feel strongly.

Reply:
I'm not strongly opinionated enough to block this change, but FWIW the term "fbs" requires me to go read the code to understand how the variable is actually used.

It's also kind of a misnomer: "fbs" could mean easily be interpreted as *all* field behaviors, not just required ones.